patches/redirectToBoard.js

```diff
@@ -0,0 +1,62 @@
+const isEduVulcan = !!window.location.hostname.match(/^((?!dziennik).)*$/);
+const isMobile = window.innerWidth < 1024;
+
+const getLogoElement = () =>
+    document.querySelector(
+        `${
+            isEduVulcan
+                ? ".header__logo-product a"
+                : ".header__logo-product img"
+        }`,
+    );
+
+const logoObserver = new MutationObserver((mutationsList, observer) => {
+    if (getLogoElement()) {
+        observer.disconnect();
+        redirectToBoard();
+    }
+});
+
+function redirectToBoard() {
+    const logoElement = getLogoElement();
+    if (!!window.location.hostname.match(/^(dziennik-)?wiadomosci.*/)) {
+        const url = !!window.location.hostname.match(
```

Comment:
two times checked for the same thing and still adding 2nd option that isn't even possible